src/index.ts

```diff
@@ -265,6 +266,11 @@ export async function signAddon({
       );
     }
   }
+  if (!output && channel === 'listed') {
```

Comment:
Same issue here, can we throw an error here? or at least set the `exitCode` to non-zero so that the next task will know that we don't get a file downloaded.

Reply:
Thing is, in my CI I'm submitting an add-on that needs manual review. I don't think it's right for the CI to fail if manual review is needed.

Reply:
> Same issue here, can we throw an error here? or at least set the `exitCode` to non-zero so that the next task will know that we don't get a file downloaded.

Do you think we could add a flag to toggle the behavior of silently not downloading a file? What should it be named in the function and in the CLI?

Reply:
I see, that makes sense. Now I got why you proposed to return immediately when `output` is empty.

Reply:
> Do you think we could add a flag to toggle the behavior of silently not downloading a file?

Yes, that would be a good choice. But since we know what we passed, it's also acceptable to handle the output differently by ourselves as the users.